<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerkkolaskuMaaritys</class>
 <widget class="QWidget" name="VerkkolaskuMaaritys">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_5">
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 0, 125);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Tämän version verkkolaskuominaisuus ei ole yhteensopiva nykyisen verkkolaskustandardin kanssa. &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Verkkolasku on käytettävissä uudemman version (Kitsas) lisäominaisuutena.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Oma verkkolaskuosoite</string>
       </property>
       <property name="buddy">
        <cstring>osoiteEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="osoiteEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Verkkolaskuoperaattorin välittäjätunnus</string>
       </property>
       <property name="buddy">
        <cstring>valittajaEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="valittajaEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Verkkolaskujen tallennuskansio</string>
       </property>
       <property name="buddy">
        <cstring>kansioEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="kansioEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="kansioNappi">
         <property name="text">
          <string>Valitse</string>
         </property>
         <property name="icon">
          <iconset resource="../pic/pic.qrc">
           <normaloff>:/pic/avaatiedosto.png</normaloff>:/pic/avaatiedosto.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="soapBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Kirjoita SOAP-kehys</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/paketti2.png</normaloff>:/pic/paketti2.png</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="pdfBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Liitä pdf-lasku</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/pdf.png</normaloff>:/pic/pdf.png</iconset>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="zipBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Tee zip-paketti</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/paketti.png</normaloff>:/pic/paketti.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pdfBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>zipBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
